projects/components/src/table/header/table-header-cell-renderer.component.ts

```diff
@@ -102,9 +119,14 @@ export class TableHeaderCellRendererComponent implements OnInit, OnChanges {
 
   public isFilterable: boolean = false;
 
+  @ViewChild('htmlTooltip')
+  public htmlTooltipTemplate?: TemplateRef<unknown>;
+  public sanitizedHtmlForTooltip?: string | null;
```

Comment:
let's keep this as `?: string` (undefined or string) - we don't want to work with nulls, so wherever angular may return one, throw a `?? undefined` on.

Reply:
Done